cmd/root.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"drupal9ci/scripts"
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "drupal9ci",
+	Short: "A brief description of your application",
+	Long: `A longer description that spans multiple lines and likely contains
+examples and usage of using your application. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	// Uncomment the following line if your bare application
+	// has an action associated with it:
+	// Run: func(cmd *cobra.Command, args []string) { },
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute(setupScripts *scripts.SetupScripts) {
+	var selectedCIProvider string
+	var setupScript string
+	var err error
+
+	if len(os.Args) > 1 {
```

Comment:
Had to add this in order to set a CI provider at the CircleCI job that tests the application.